### ISSUE TITLE
Motion Alarm Unknown

### ISSUE THREAD

**Issue by ilhavo** [dbuezas/icsee-ptz#32]
Add my "Bresder 4k Camera" to integration and all good, but motion alarm appear "unknown"..
How fix it?
Thank you

**Comment by dbuezas** [comment]
Enable alarms first in the original app

**Comment by ilhavo** [comment]
Its activated...
Camera need see moviment first for detect sensor in HA...
All good now, but ptz wont work.. :(
When press right ir left, nothing...

**Comment by dbuezas** [comment]
Yes that's normal, take a thorough look at the readme and it should be clear

**Comment by ilhavo** [comment]
when call service...

icsee_ptz.move. must contain at least one of entity_id, device_id, area_id.. Got {'type': 'execute_script', 'sequence': [{'service': 'icsee_ptz.move', 'target': {}, 'data': {'cmd': 'DirectionUp', 'step': 2, 'preset': 0}}], 'id': 96}

**Comment by dbuezas** [comment]
See https://github.com/dbuezas/icsee-ptz?tab=readme-ov-file#example-automation

The entity is the same you mentioned above.

**Comment by j-augustyn** [comment]
Hello,
i have an old iCSee solar camera. On the android app i set the motion alarm to on. When I move, I get a notification from the app but in HA it is still "unknow". Do you have any solution for this ?
V1.01.LITEOS.000729MG.00000.030007.00000
